streaming/src/channel.h

```diff
@@ -165,6 +171,7 @@ class MockConsumer : public ConsumerChannel {
                                           uint64_t checkpoint_offset) override {
     return StreamingStatus::OK;
   }
+  StreamingStatus RefreshChannelInfo() {}
```

Comment:
Minconsumed id can be fetched from static map?
Should we implement it here for unittest?

Reply:
> Minconsumed id can be fetched from static map?
> Should we implement it here for unittest?

If we want to mock and test `RefreshChannelInfo` in MockQueue, ChannelInfo should be added in MockQueue.